projects/go-lib/src/lib/components/go-select/go-select.component.ts

```diff
@@ -10,6 +10,7 @@ import { FormControl } from '@angular/forms';
 export class GoSelectComponent implements OnInit {
   id: string;
 
+  @Input() appendTo: string;
```

Comment:
@jaredami do we need a default here?

Reply:
How does this work if we pass undefined to `ng-select`?

Reply:
Good question. From [the docs:](https://ng-select.github.io/ng-select#/append-to-element) "By default dropdown panel is rendered as child element of ng-select, but you can append dropdown to any element using appendTo." 

If we were to set a default value, I'm not sure what it should be so that it would match the default behavior.

At any rate, I tested it with passing nothing in as well explicitly passing in `undefined`, and in both cases it appears to behave the same as the default. So it seems to work fine without explicitly naming a default. Am I missing something?

Reply:
@jaredami nah, sounds good to me.